NCAR/ufs-weather-model
# Update dtc/develop from NOAA-EMC develop 2020/07/21, prepare for SRW App

Update dtc/develop from NOAA-EMC develop 2020/07/21, and prepare for SRW App release. Changes in this PR:

- Tom Black's dycore regional DA fix
- Add flake model
- Allow multiple regtest ecflow suites to run on the same server
- Add PR, Issues for bugfix and feature request templates
- Remove regression tests fv3_ccpp_gsd_sar_v1 and fv3_ccpp_rrfs_v0, add regression test fv3_ccpp_rrfs_v1beta_debug
- Update several regression test configuration files (for cheyenne, orion, ...)

Associated PRs:

https://github.com/NCAR/GFDL_atmos_cubed_sphere/pull/16
https://github.com/NCAR/ccpp-framework/pull/313
https://github.com/NCAR/ccpp-physics/pull/479
https://github.com/NCAR/fv3atm/pull/61
https://github.com/NCAR/ufs-weather-model/pull/62

For regression testing, see below.
